core/mmu.cpp

```diff
@@ -36,6 +56,9 @@ u16 MMU::read16Bit(u16 address) {
         std::cout << "ERROR: Attempted read from forbidden address: " << address << std::endl;
         return 0x00;
     }
+    if (blockedByPPU(address)) {
+        return 0xFF;
```

Comment:
This is `read16Bit()`, so this should return `0xFFFF`

Reply:
> Was the issue `read()` and `write()`?

That would have been the issue had I implemented the blocking modes while I was testing this. The main issue was a bug in my logic where I was accidentally adding an offset to the value at a memory address rather than the offset to the memory address itself

Reply:
I don't know why that replied to this thread... haha but good catch on the read16Bit!